### PR TITLE
ATO-979/d: Get verifiedMfaMethodType from Orch session instead of shared

### PR DESCRIPTION
**Pull request by billrdunn**
## What

Now that `verifiedMfaMethodType` is passed as a claim in the userinfo response from Auth and stored in the Orch session, we can get and use the value from the Orch session instead of the shared session.

## Testing
This needs to be tested in staging as it affects IpvCallback and AuthCode lambdas.

### CODE REVIEW

File: oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthCodeHandler.java [hunk 1]
```diff
@@ -173,6 +173,9 @@ public APIGatewayProxyResponseEvent authCodeRequestHandler(
             return generateApiGatewayProxyErrorResponse(e.getStatusCode(), e.getErrorResponse());
         }
 
+        LOG.info("orchSession: {}", orchSession);
```
Comment (GHSwallow) [inline oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthCodeHandler.java:176]: Looks like your temp logging for testing is still here

File: ipv-api/src/main/java/uk/gov/di/authentication/ipv/lambda/IPVCallbackHandler.java [hunk 2]
```diff
@@ -357,6 +365,10 @@ public APIGatewayProxyResponseEvent handleRequest(
             var vtrList = clientSession.getVtrList();
             var userIdentityError =
                     ipvCallbackHelper.validateUserIdentityResponse(userIdentityUserInfo, vtrList);
+            LOG.info("orchSession: {}", orchSession);
```
[the remark this answers is not in the document]
Comment (billrdunn) [reply ipv-api/src/main/java/uk/gov/di/authentication/ipv/lambda/IPVCallbackHandler.java:368]: ooh yeah good spot!

File: oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthCodeHandler.java (hunk 1 above)
Comment (billrdunn) [reply oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthCodeHandler.java:176]: I can't test in dev so I was going to pause promotions from staging to test in staging, then remove the logging

File: ipv-api/src/main/java/uk/gov/di/authentication/ipv/lambda/IPVCallbackHandler.java [hunk 3]
```diff
@@ -223,6 +224,13 @@ public APIGatewayProxyResponseEvent handleRequest(
                             .getSession(sessionCookiesIds.getSessionId())
                             .orElseThrow(
                                     () -> new IPVCallbackNoSessionException("Session not found"));
+            OrchSessionItem orchSession =
```
[the remark this answers is not in the document]
Comment (billrdunn) [reply ipv-api/src/main/java/uk/gov/di/authentication/ipv/lambda/IPVCallbackHandler.java:227]: It's a bit hidden: `orchSession` gets passed to `ipvCallbackHelper.generateReturnCodeAuthenticationResponse` and then `authCodeResponseService.getDimensions` where we do `orchSession.getVerifiedMfaMethodType()`

File: ipv-api/src/main/java/uk/gov/di/authentication/ipv/lambda/IPVCallbackHandler.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (billrdunn) [reply ipv-api/src/main/java/uk/gov/di/authentication/ipv/lambda/IPVCallbackHandler.java]: fixed

File: ipv-api/src/main/java/uk/gov/di/authentication/ipv/lambda/IPVCallbackHandler.java (hunk 3 above)
[the remark this answers is not in the document]
Comment (GHSwallow) [reply ipv-api/src/main/java/uk/gov/di/authentication/ipv/lambda/IPVCallbackHandler.java:227]: ah yea I see, you use it in a later commit 👍

File: oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthCodeHandler.java (hunk 1 above)
Comment (GHSwallow) [reply oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthCodeHandler.java]: Ah gotcha. I'm inclined to include TODO comments to make it clear (mainly to myself) they're to be removed soon, but that's just what's helpful to me, and these will be removed immediately, so not a problem for anyone else seeing these 👍

**Review by GHSwallow — APPROVED** [review]

**Pull request merged**